ee/clickhouse/sql/sessions/no_events.py

```diff
@@ -42,6 +42,16 @@
                 {date_from}
                 {date_to} 
                 {filters}
+                AND distinct_id IN (
+                    SELECT distinct distinct_id
+                    FROM
+                        events
+                    WHERE team_id = %(team_id)s
+                    {date_from}
+                    {date_to}
+                    ORDER BY timestamp DESC
+                    {sessions_limit}
+                )
```

Comment:
Ah cool i see. This query specifically isn't used in the session_list query though so you'll have to add it to `list.py` for it to take effect

Reply:
no_events should remain unchanged because for trend metrics it should grab as much as possible

Reply:
whoopsy, done